common/changes/office-ui-fabric-react/autocompleteBool_2017-10-05-17-09.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Make combo box autocomplete prop a boolean",
+      "type": "patch"
```

Comment:
This isn't a patch, it's a breaking change. @jspurlin, why was this a string in the first place?

Reply:
Originally it was a boolean, but was changed to align with HtmlAttributes  so that other html props could be passed in. See: https://github.com/OfficeDev/office-ui-fabric-react/commit/21c744979934285a8759d93ce8cd8748a16053a5